modules/appeals_api/spec/support/doc_helpers.rb

```diff
@@ -149,8 +150,14 @@ def self.use_shared_schemas?
     !DocHelpers.decision_reviews?
   end
 
+  def self.running_rake_task?
+    # SWAGGER_DRY_RUN is set in the appeals rake tasks: if it's not set, it means the spec is running as part of
+    # a normal rspec suite instead.
+    ENV['SWAGGER_DRY_RUN'].present?
```

Comment:
I added this helper because when these tests run under rspec alongside other specs (that is, when you run `bundle exec rspec` instead of using the docs rake tasks like `bundle exe rake rswag:appeals_api:all` etc.), the environment is not configured in any particular way, so the API_NAME and API_VERSION environment variables can't be correct for all specs.

I used this helper to entirely skip the appeals status doc specs when running under rspec. In cases where we're actually running rake tasks to generate docs, the environment variables are still used as before.